context/app/api/client.py

```diff
@@ -165,45 +238,99 @@ def get_provenance(self, uuid):
             token = 'fake-token'
-        else:
-            # TODO: Hit File API
-            cellsUri = 'https://assets.test.hubmapconsortium.org/' \
-                '686cd8e0c2a9fa2dc1a321330158dcd7/umap/' \
-                'cluster_marker_genes/cluster_marker_genes.json'
-            token = self.nexus_token
-        return {
-            "description": "DEMO",
-            "layers": [
-                {
-                    "name": "cells",
-                    "type": "CELLS",
-                    "url": cellsUri,
-                    "requestInit": {
-                        "headers": {
-                            'Authorization': 'Bearer ' + token
-                        }
-                    }
-                },
-            ],
-            "name": "Linnarsson",
-            "staticLayout": [
-                {
-                    "component": "scatterplot",
-                    "props": {
-                        "mapping": "UMAP",
-                        "view": {
-                            "zoom": 4,
-                            "target": [0, 0, 0]
+            return {
+                'description': 'DEMO',
+                'layers': [
+                    {
+                        'name': 'cells',
+                        'type': 'CELLS',
+                        'url': cellsUri,
+                        'requestInit': {
+                            'headers': {
+                                'Authorization': 'Bearer ' + token
+                            }
                         }
                     },
-                    "x": 0, "y": 0, "w": 12, "h": 2
-                },
-            ]
+                ],
+                'name': 'Linnarsson',
+                'staticLayout': [
+                    {
+                        'component': 'scatterplot',
+                        'props': {
+                            'mapping': 'UMAP',
+                            'view': {
+                                'zoom': 4,
+                                'target': [0, 0, 0]
+                            }
+                        },
+                        'x': 0, 'y': 0, 'w': 12, 'h': 2
+                    },
+                ]
+            }
+        else:
+            return self._build_vitessce_conf(entity)
+
+    def _build_assets_url(self, rel_path, uuid):
+        base_url = urllib.parse.urljoin(
+            current_app.config['ASSETS_ENDPOINT'],
+            str(Path(uuid) / Path(rel_path))
+        )
+        token_param = urllib.parse.urlencode({'token': self.nexus_token})
+        return base_url + '?' + token_param
+
+    def _build_image_schema(self, rel_path, uuid):
+        schema = {}
+        schema['name'] = Path(rel_path).name
+        schema['type'] = 'ome-tiff'
+        schema['url'] = self._build_assets_url(rel_path, uuid)
+        schema['metadata'] = {}
+        offsets_path = Path(CODEX_OFFSETS_PATH) / \
+            Path(Path(rel_path).name.replace('ome.tiff', 'offsets.json'))
```

Comment:
```suggestion
            rel_path.replace('ome.tiff', 'offsets.json')
```
Assuming `rel_path` is a string.

Reply:
Path extracts the name.